_newsletters/2019-01-01-january-2019.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: January 2019 Newsletter
+category: newsletters
+last_modified_at: 
+tags: newsletters
+newsletter-year: 2019
+sidenav: about_us
+---
+
+## SERVICE STATUS
+
+### Our lights are on, but we won't be here
+
+Both the Search Admin Center and your search results pages will remain up and running during the temporary government shutdown. But, we are out of the office and won’t respond to your emails or calls. Contractors will respond to any service disruptions.
+
+You can continue to send us your general requests and questions, and we'll respond when we return to the office.
+
+If you’re experiencing an emergency and need immediate assistance during the shutdown, send an email to <a href="mailto:search@gsa.gov">search@gsa.gov</a> including the word “URGENT” in the subject line.
+
+### How to tell searchers about your site's status
+
+Is your website affected by the shutdown? Below are two ways you can post a message on your search results page to tell searchers about your site’s status.
+
+1. Post a search page alert.
+
+    You can add a message to the top of your search results. Searchers will always see your message, regardless of their search term.
```

Comment:
Is this indent ok? and same question for lines 29, 33, 35, and 37.

Reply:
yep : https://cg-a72b65a7-73f0-4859-a1d7-dc953cf6ade8.app.cloud.gov/preview/gsa/search-gov-website/header-newsletter-updates/about/updates/newsletters/january-2019.html